examples/xla_step.py

```diff
@@ -40,7 +40,7 @@ def actor_step(iter, loop_var):
     handle0, states = loop_var
     action = policy(states)
     if is_legacy_gym:
-      handle1, (new_states, rew, done, info) = step(handle0, action)
+      handle1, (new_states, rew, done, info, _) = step(handle0, action)
```

Comment:
I think this change is wrong

Reply:
maybe it's a version issue, I got it running with this change, you can change it back.